src/components/EWNRSSTable.tsx

```diff
@@ -0,0 +1,152 @@
+  let items = filtered.slice(firstIndex, lastIndex);
+
+  const clickOrdering = (header: string) => {
+    if (ordering === header) {
+      setOrdering(`-${header}`);
+    } else {
+      setOrdering(header);
+    }
+    setPage(0);
+  };
+
+  const isOrdering = (header: string) => {
+    if (ordering === header) {
+      return <>&#x25B2;</>;
+    } else if (ordering === `-${header}`) {
+      return <>&#x25BC;</>;
+    } else {
+      return null;
+    }
+  };
+
+  return (
+    <div style={{ padding: "1rem" }}>
+      <select
+        onChange={(e) => {
+          setFilter(e.target.value);
+          setPage(0);
+        }}
+      >
+        <option value={""}>-- Filter on warning area --</option>
+        {areaNames.map((areaName) => (
+          <option key={areaName} value={areaName}>
+            {areaName}
+          </option>
+        ))}
+      </select>
+      <div className={styles.WarningsTable}>
+        <div className={styles.tr}>
+          <div className={styles.thtd}>Publication date</div>
+          <div className={styles.thtd} onClick={() => clickOrdering("area")}>
+            Warning Area {isOrdering("area")}
+          </div>
+          <div className={styles.thtd} onClick={() => clickOrdering("level")}>
+            Trigger level {isOrdering("level")}
+          </div>
+          <div className={styles.thtd}>Link</div>
+        </div>
+        {items.map(({ pubDate, area, warning, link }) => (
+          <div className={styles.tr} key={link}>
+            <div
+              className={styles.tdLeft}
+              style={{ overflow: "hidden", textOverflow: "ellipsis", whiteSpace: "nowrap" }}
+            >
+              {pubDate}
+            </div>
+            <div
+              className={styles.tdLeft}
+              style={{ overflow: "hidden", textOverflow: "ellipsis", whiteSpace: "nowrap" }}
+            >
+              {area}
+            </div>
+            <div
+              className={styles.tdLeft}
+              style={{ overflow: "hidden", textOverflow: "ellipsis", whiteSpace: "nowrap" }}
+            >
+              {warning}
+            </div>
+            <div
+              className={styles.tdLeft}
+              style={{ overflow: "hidden", textOverflow: "ellipsis", whiteSpace: "nowrap" }}
+            >
+              <a href={link} target="_new">
+                click <BiLinkExternal />
+              </a>
+            </div>
+          </div>
+        ))}
+      </div>
+      <p className={styles.numWarnings}>
+        Showing warnings {firstIndex + 1} to {lastIndex} of {numTriggers}.
+      </p>
+      <input
+        type="button"
+        disabled={!previousEnabled}
+        value="<"
+        onClick={() => previousEnabled && setPage(page - 1)}
+      />
+      <input
+        type="button"
+        disabled={!nextEnabled}
+        value=">"
+        onClick={() => nextEnabled && setPage(page + 1)}
+      />
```

Comment:
I would prefer button over input type=button.
But it is not a big deal.
https://stackoverflow.com/questions/469059/button-vs-input-type-button-which-to-use

Reply:
Will fix